content/en/docs/concepts/services-networking/_index.md

```diff
@@ -7,44 +7,195 @@ description: >
-much like VMs or physical hosts from the perspectives of port allocation, 
-naming, service discovery, [load balancing](/docs/concepts/services-networking/ingress/#load-balancing), 
-application configuration, and migration.
-
-Kubernetes imposes the following fundamental requirements on any networking
+The Kubernetes network model enables container networking within a Pod and between Pods on the same or different nodes. Figure 1 depicts a cluster with a control plane and a couple of nodes, each with Pods containing one more containers. It points to the location in your cluster where the basic container network functions apply.
+
+{{< figure src="/docs/images/k8s-net-model-intro2.drawio.svg" alt="diagram for k8net model intro showing network function location" class="diagram-large" caption="Figure 1. K8s cluster showing where container network functions are performed" >}}
+
+In the single node (Node 1) case, you have connectivity between containers running on a single Pod, and between containers running on two or more Pods on the same node. 
+
+In the multi-node case (Node 1 and Node 2), you have container communications between Pods on nodes connected via a network(s). In this case you will require network functions on both nodes.
+
+You can look over container networking functions described in the _examples_ below.
+
+* [Containers on the same pod](#containers-on-the-same-pod)
+* [Pods on the same node](#pods-on-the-same-node)
+* [Pods on different nodes using a physical network](#pods-on-different-nodes-using-a-physical-network)
+* [Pods on different nodes using a virtual overlay network](#pods-on-different-nodes-using-a-virtual-overlay-network)
+
+{{< note >}}
+The container networking _examples_ shown below do not imply or describe a specific implementation. They are simply examples used to illustrate the container network functions defined in the K8s network model.  
+{{< /note >}}
+
+## Requirements
+
+Every [Pod](/docs/concepts/workloads/pods/) in your cluster gets its own unique cluster-wide IP address called a _pod IP_. 
+
+If you have deployed an [IPv4/IPv6 dual stack](/docs/concepts/services-networking/dual-stack/) cluster,
+then you - or your network plugin(s) - must allocate pod IPs for IPv4 and IPv6 for each pod.
+
+This is performed per [_address family_](https://www.iana.org/assignments/address-family-numbers/address-family-numbers.xhtml), with one for IPv4 addresses and one for IPv6 addresses. You can assign an Ethernet MAC address to each Pod. Kubernetes does not require that a Pod has a unique identity at the data-link layer.
+
+Kubernetes imposes the following requirements on any networking
 implementation (barring any intentional network segmentation policies):
 
-   * pods can communicate with all other pods on any other [node](/docs/concepts/architecture/nodes/) 
-     without NAT
-   * agents on a node (e.g. system daemons, kubelet) can communicate with all
```

Comment:
this is important, is reflected in other place?

Reply:
Yes but I'll put the "agents on a node" bullet back in.